fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/RuleSet.java

```diff
@@ -233,13 +230,7 @@ public class RuleSet {
                 RemoteScanPartitionPruneRule.ICEBERG_SCAN,
                 RemoteScanPartitionPruneRule.DELTALAKE_SCAN,
                 RemoteScanPartitionPruneRule.FILE_SCAN,
-                PushDownMinMaxConjunctsRule.HIVE_SCAN,
-                PushDownMinMaxConjunctsRule.HUDI_SCAN,
-                PushDownMinMaxConjunctsRule.ICEBERG_SCAN,
-                PushDownMinMaxConjunctsRule.DELTALAKE_SCAN,
-                PushDownMinMaxConjunctsRule.FILE_SCAN,
-                new EsScanPartitionPruneRule(),
-                new PartitionPredicatePrune()
```

Comment:
why you remove these rules? you move them into OptExternalPartitionPruner?

Reply:
yep.